### PR TITLE
feat: knowledgebase support more granular update

### PR DESCRIPTION
**Pull request by Abirdcfly**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it
With `pgvector` support, we can perform more fine-grained operations, such as incrementally updating vector data in behavioral units, which this pr implements. Note that after updating `qa.csv` in-place, you need to modify the annotation of knowlegebase to trigger the reconcile : `arcadia.kubeagi.k8s.com.cn/update-source-file-time`

#### Which issue(s) this PR fixes

Fixes #

### CODE REVIEW

File: deploy/charts/arcadia/values.yaml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 global:
   oss:
     bucket: &default-oss-bucket "arcadia"
+  ## @param global.defaultVectorStoreType Defines the default vector database type, currently `chroma` and `pgvector` are available
+  ## When the option is `chroma`, it needs `chromadb.enabled` to be `true` as well to work.
+  ## When the option is `pgvector`, it needs `postgresql.enabled` to be `true` as well to work.
+  defaultVectorStoreType: pgvector
```
Comment (nkwangleiGIT) [inline deploy/charts/arcadia/values.yaml:7]: chromadb is still enabled for now, can we update it to false if we'll use pgvector by default now?

File: tests/deploy-values.yaml [hunk 2]
```diff
@@ -1,3 +1,5 @@
+global:
+  defaultVectorStoreType: pgvector
```
Comment (nkwangleiGIT) [inline tests/deploy-values.yaml:2]: chromadb is still enabled for now, can we update it to false if we'll use pgvector by default now?

File: deploy/charts/arcadia/values.yaml (hunk 1 above)
Comment (Abirdcfly) [reply deploy/charts/arcadia/values.yaml:7]: We have 2 separate configurations: `chromadb.enabled` and `postgresql.enable` which control the startup of chroma and postgresql respectively, and I think it's possible to change `chromadb.enabled` to `false` by default from the current version onwards.Any ideas? @nkwangleiGIT @bjwswang @0xff-dev

Comment (bjwswang) [reply deploy/charts/arcadia/values.yaml:7]: @Abirdcfly  agree. We can focus on pgvector. 

Set chroma to false by default. 

Besides.. if chroma or pg enbaled,we should create a CR vectorstore as well.

File: deploy/charts/arcadia/Chart.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: arcadia
 description: A Helm chart(KubeBB Component) for KubeAGI Arcadia
 type: application
-version: 0.2.9
+version: 0.2.10
```
Comment (bjwswang) [inline deploy/charts/arcadia/Chart.yaml:5]: version bump

**Review by bjwswang — APPROVED** [review]

**Pull request merged**